tests/tensorflow_autolog/test_tensorflow_autolog.py

```diff
@@ -423,16 +480,10 @@ def test_tf_estimator_autolog_model_can_load_from_artifact(tf_estimator_random_d
                                          "/model", session)
 
 
-@pytest.fixture
-def duplicate_autolog_tf_estimator_run(tmpdir, manual_run, export):
-    mlflow.tensorflow.autolog(every_n_iter=23)  # 23 is prime; no false positives in test
-    run = tf_estimator_random_data_run(tmpdir, manual_run, export)
```

Comment:
This call to a fixture as a function is invalid. Interestingly, `pytest` did not pick up this issue until implicit `end_run()` calls were added to the new `tf_keras` tests. I've removed this fixture since it's only used in a single test case, and I've modified the test case to call `autolog(every_n_iter=23)` so that the same state is achieved.